### ISSUE TITLE
Add first name as a new field

### ISSUE THREAD

**Issue by paulinebessoles** [PopulateTools/decidim-module-extra_user_fields#12]
Hi @ferblape! Happy new year 😁 

Would you accept a contribution to add a field for the user first name?

**Comment by ferblape** [comment]
Yes, it'll make the module more complete. I still have to update and merge https://github.com/PopulateTools/decidim-module-extra_user_fields/pull/6, which add a couple of new fields too

**Comment by paulinebessoles** [comment]
Thank you for your quick answer! I'll keep you updated when we have some time to send the new first name field.
